apps/aecore/test/aec_trees_tests.erl

```diff
@@ -81,3 +81,132 @@ make_spend_tx(Sender, Recipient) ->
                                        nonce => 1,
                                        payload => <<>>}),
     SpendTx.
+
+poi_test_() ->
```

Comment:
It would be nice to have a `poi_serialization_test_` exemplifying the serialized PoI up to nibble level. This comes mostly from https://github.com/aeternity/protocol/pull/106#discussion_r191706391
Though totally optional for this PR.